tests/mypy_test.py

```diff
@@ -206,6 +262,8 @@ def main():
                     mypy_main("", sys.stdout, sys.stderr)
             except SystemExit as err:
                 code = max(code, err.code)
+            finally:
```

Comment:
Why not put all the code in the `with NamedTemporaryFile` block so you don't need to explicitly delete the file?

Reply:
Out of the tempfile python docs:
Whether the name can be used to open the file a second time, while the named temporary file is still open, varies across platforms (it can be so used on Unix; it cannot on Windows NT or later).

Good thing I develop on windows, or I'd never have guessed.

Reply:
I just stumbled onto https://bugs.python.org/issue14243 too, sounds painful.